packages/checkboxInput/tests/CheckboxInput.test.tsx

```diff
@@ -10,39 +9,39 @@ expect.addSnapshotSerializer(createSerializer());
 describe("CheckboxInput", () => {
   it("renders all appearances", () => {
     Object.keys(InputAppearance).forEach(appearance => {
-      const component = mount(
+      const { asFragment } = render(
         <CheckboxInput
           id="defaultId"
           inputLabel="Sample label"
           value="default"
           appearance={InputAppearance[appearance]}
         />
       );
-      expect(toJson(component)).toMatchSnapshot();
+      expect(asFragment()).toMatchSnapshot();
     });
   });
 
   it("renders with a hidden label", () => {
-    const component = mount(
+    const { asFragment } = render(
       <CheckboxInput
         id="hiddenLabel"
         inputLabel="Sample Label"
         value="default"
         showInputLabel={false}
       />
     );
-    expect(toJson(component)).toMatchSnapshot();
+    expect(asFragment()).toMatchSnapshot();
   });
 
   it("renders indeterminate", () => {
-    const component = mount(
+    const { asFragment } = render(
       <CheckboxInput
         id="indeterminateId"
         inputLabel="Sample Label"
         value="indeterminate"
         indeterminate={true}
       />
     );
-    expect(toJson(component)).toMatchSnapshot();
+    expect(asFragment()).toMatchSnapshot();
```

Comment:
okay so which should we do, testing-library's render or react's create?

Reply:
The `ButtonBase` test is failing with `create`. We get `this.container.className`. A lot of people lean towards using `create`, but we might have to use `asFragment`. We can refactor the other tests to use `asFragement` as we move forward with removing enzyme. https://stackoverflow.com/a/58664924/3058839